archivers/brotli/Portfile

```diff
@@ -26,4 +26,14 @@ checksums           rmd160  3e2402d137fd75f898007d3730773b32025a4857 \
 
 patchfiles          no-runtime-linker-path-in-pc-files.patch
 
+destroot {
```

Comment:
This should be a `post-destroot` block. Like this, normal destroot is not run and we will end up with the man pages only.

Reply:
Thanks!